Automattic/wp-calypso
# Split clean:public into multiple targets.

This helps avoid argument list length errors caused internally in `rimraf` or due to shell glob expansion.

#### Changes proposed in this Pull Request

* Split `clean:public` into multiple targets.
* Add quotes to globs in the various `clean:public*` targets.

#### Testing instructions

* Run `npm run distclean` and ensure there are no errors.
* Run `npm run start` and ensure there are no errors.
